MantidQt/CustomInterfaces/src/Tomography/TomoToolConfigDialogBase.cpp

```diff
@@ -0,0 +1,39 @@
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogBase.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogTomoPy.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogAstra.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogSavu.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogCustom.h"
+
+#include "MantidKernel/make_unique.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+std::unique_ptr<TomoToolConfigDialogBase>
+TomoToolConfigDialogBase::getToolDialogFor(const std::string &toolName) {
+
+  if (toolName == "TomoPy") {
+    return Mantid::Kernel::make_unique<TomoToolConfigDialogTomoPy>();
+  }
+  if (toolName == "Astra") {
```

Comment:
Can we bring the separate if statements together into an if, else-if, else. So its obvious that its only meant to go into one branch?